modules/mobile/spec/requests/mobile/v0/claims/decision_letters_spec.rb

```diff
@@ -38,10 +42,9 @@
     context 'with a valid response' do
       context 'with mobile_filter_doc_27_decision_letters_out flag enabled' do
         it 'returns expected decision letters' do
-          Flipper.enable('mobile_filter_doc_27_decision_letters_out')
+          Flipper.enable(:mobile_filter_doc_27_decision_letters_out)
```

Comment:
I know these were already like this, but we have new conventions for Flipper specs. Do you mind updating these in accordance please?
https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#FeatureTogglesGuide-Backendexample
Particularly this part:
```
      # Recommended to limit the feature flag state to the scope of this test
      allow(Flipper).to receive(:enabled?).with(:feature_flag).and_return(true)
      # Not recommended. Affects global state and can have unintended side effects when tests run in parallel
      Flipper.enable(:feature_flag)
```